money.go

```diff
@@ -18,9 +18,13 @@ type Money struct {
 
 // New creates and returns new instance of Money
 func New(amount int64, code string) *Money {
+	c := currencies[code]
```

Comment:
What do you mean by these 4 lines? 
If you want to create "Fallback" currency, method `get` will do it for you if currency is not found.